php/WP_CLI/Runner.php

```diff
@@ -1163,7 +1163,7 @@ private function setup_bootstrap_hooks() {
 		// In a multisite install, die if unable to find site given in --url parameter
 		if ( $this->is_multisite() ) {
 			WP_CLI::add_wp_hook( 'ms_site_not_found', function( $current_site, $domain, $path ) {
-				WP_CLI::error( "Site {$domain}{$path} not found." );
+				WP_CLI::error( "Site {$domain}{$path} not found. Check value of 'DOMAIN_CURRENT_SITE' into wp-config.php." );
```

Comment:
This can also be triggered if `DOMAIN_CURRENT_SITE` is set, but the database did not contain the specific site that has been targeted (not set up yet, error in database, ...)